src/Microsoft.Graph.Core/Microsoft.Graph.Core.csproj

```diff
@@ -19,10 +19,10 @@
     <DelaySign>false</DelaySign>
     <AssemblyOriginatorKeyFile>35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-    <VersionPrefix>1.22.0</VersionPrefix>
+    <VersionPrefix>1.23.0</VersionPrefix>
     <VersionSuffix></VersionSuffix>
-    <PackageReleaseNotes>- BaseRequestBuilder is now an IBaseRequestBuilder.
-- Requests that return an OData primitive now work as expected. 
+    <PackageReleaseNotes>
+      - Fix for #120: Exception thrown when slice size is not specified for LargeFileUploadTask 
```

Comment:
nit: Align to the left margin as the tab will make it look like https://www.nuget.org/packages/Microsoft.Graph.Core/1.21.0

Reply:
Thanks Michael. Actioned and resolved